src/commcare_cloud/ansible/roles/formplayer/tasks/main.yml

```diff
@@ -101,14 +101,40 @@
     state: directory
     owner: "{{ cchq_user }}"
     group: "{{ cchq_user }}"
-    mode: 0755
+    mode: "0755"
   tags:
     - localsettings
     - formplayer_deploy
   with_items:
     - "{{ formplayer_build_dir }}"
     - "{{ [] if _should_update_formplayer_in_place else formplayer_release_dir }}"
+
+- name: Check if formplayer data dir exists
+  stat:
+    path: "{{ formplayer_data_dir }}"
+  tags:
+    - localsettings
+    - formplayer_deploy
+  register: check_data_dir
+
+- name: Formplayer data dir
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: "{{ cchq_user }}"
+    group: "{{ cchq_user }}"
+    mode: "0755"
+  tags:
+    - localsettings
+    - formplayer_deploy
+  with_items:
     - "{{ formplayer_data_dir }}"
+  when: not check_data_dir
```

Comment:
@shyamkumarlchauhan why is this necessary? Ansible should be handling this automatically.

Reply:
as this will be a NFS mount so ownership/mode change of a NFS mount may generate error if cchq_uid/cchq_gid is not defined in public.yml